webapp-django/crashstats/supersearch/static/supersearch/js/socorro/date_filters.js

```diff
@@ -51,10 +51,15 @@ $(function () {
             var value = parseInt(range.slice(0, -1));
             var unit = range.slice(-1);
             var toDate = moment().utc().toDate();
+            var toDateString = moment(toDate).utc().format("MMM DD, YYYY - HH:mm");
             var fromDate = moment().utc().subtract(value, unit).toDate();
+            var fromDateString = moment(fromDate).utc().format("MMM DD, YYYY - HH:mm");
 
-            setDate('to', toDate);
-            setDate('from', fromDate);
+            filters.to.set('minDate', fromDate);
+            filters.from.set('maxDate', toDate);
+            removeSelectedShortcut();
+            filters.to.altInput.value = toDateString;
```

Comment:
This works, but isn't documented (vs the config values for `altInput`, `altFormat`, etc.). We probably should avoid using undocumented properties whenever possible so that they don't change out from underneath us when we upgrade the library.

Reply:
> We probably should avoid using undocumented properties

You're right. Let me find another way.